projects/plugins/jetpack/_inc/client/recommendations/prompts/product-suggestions/index.jsx

```diff
@@ -1,134 +1,94 @@
 /**
  * External dependencies
  */
-import React, { useCallback, useEffect } from 'react';
+import React, { useMemo } from 'react';
 import { connect } from 'react-redux';
-import { Redirect } from 'react-router-dom';
-import { __, _x } from '@wordpress/i18n';
-import { ProgressBar } from '@automattic/components';
+import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import { JetpackLoadingIcon } from 'components/jetpack-loading-icon';
-import { PromptLayout } from '../prompt-layout';
-import { ProductSuggestion } from '../product-suggestion';
 import { MoneyBackGuarantee } from 'components/money-back-guarantee';
-import analytics from 'lib/analytics';
 import {
-	addSkippedRecommendation as addSkippedRecommendationAction,
 	getProductSuggestions,
-	getNextRoute,
-	isFetchingRecommendationsProductSuggestions as isFetchingSuggestionsAction,
-	isProductSuggestionsAvailable as isProductSuggestionsAvailableCheck,
-	updateRecommendationsStep as updateRecommendationsStepAction,
+	isFetchingRecommendationsProductSuggestions,
+	isFetchingRecommendationsUpsell,
 } from 'state/recommendations';
+import { isFetchingSiteDiscount, getSiteDiscount } from 'state/site/reducer';
+import { ProductSuggestion } from '../product-suggestion';
+import BackButton from '../../back-button';
+import Timer from '../../timer';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const ProductSuggestionsComponent = props => {
-	const {
-		addSkippedRecommendation,
-		nextRoute,
-		isFetchingSuggestions,
-		isProductSuggestionsAvailable,
-		updateRecommendationsStep,
-		suggestions,
-	} = props;
-
-	useEffect( () => {
```

Comment:
Though still in the `prompts` folder for now, this section is not a step of the wizard anymore. This explains the removed code.

Reply:
Yes, it's another way to look at it. I guess we can leave it in that folder then.